Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
+        let previousIndex,
             useICheck = window.SONATA_CONFIG && window.SONATA_CONFIG.USE_ICHECK
         ;
 
         // When a checkbox or an iCheck helper is clicked
-        jQuery('tbody input[type="checkbox"], tbody .iCheck-helper', subject).click(function (event) {
-            var input;
+        jQuery("tbody input[type=\"checkbox\"], tbody .iCheck-helper", subject).click(function (event) {
+
+            let input;
 
             if (useICheck) {
-                input = jQuery(this).prev('input[type="checkbox"]');
+
+                input = jQuery(this).prev("input[type=\"checkbox\"]");
+
             } else {
+
                 input = jQuery(this);
+
             }
 
             if (input.length) {
-                var currentIndex = input.closest('tr').index();
+
+                const currentIndex = input.closest("tr").index();
 
                 if (event.shiftKey && previousIndex >= 0) {
-                    var isChecked = jQuery('tbody input[type="checkbox"]:nth(' + currentIndex + ')', subject).prop('checked');
+
+                    const isChecked = jQuery(`tbody input[type="checkbox"]:nth(${currentIndex})`, subject).prop("checked");
 
                     // Check all checkbox between previous and current one clicked
-                    jQuery('tbody input[type="checkbox"]', subject).each(function (i, e) {
+                    jQuery("tbody input[type=\"checkbox\"]", subject).each((i, e) => {
+
                         if (i > previousIndex && i < currentIndex || i > currentIndex && i < previousIndex) {
+
                             if (useICheck) {
-                                jQuery(e).iCheck(isChecked ? 'check' : 'uncheck');
+
+                                jQuery(e).iCheck(isChecked ? "check" : "uncheck");
 
                                 return;
+
                             }
 
-                            jQuery(e).prop('checked', isChecked);
+                            jQuery(e).prop("checked", isChecked);
+
                         }
+
                     });
+
                 }
 
-                previousIndex  = currentIndex;
+                previousIndex = currentIndex;
+
             }
+
         });
+
     },
 
-    setup_xeditable: function(subject) {
-        Admin.log('[core|setup_xeditable] configure xeditable on', subject);
-        jQuery('.x-editable', subject).editable({
-            emptyclass: 'editable-empty btn btn-sm btn-default',
-            emptytext: '<i class="fa fa-pencil"></i>',
-            container: 'body',
-            placement: 'auto',
-            success: function(response) {
-                var html = jQuery(response);
+    "setup_xeditable" (subject) {
+
+        Admin.log("[core|setup_xeditable] configure xeditable on", subject);
+        jQuery(".x-editable", subject).editable({
+            "emptyclass": "editable-empty btn btn-sm btn-default",
+            "emptytext": "<i class=\"fa fa-pencil\"></i>",
+            "container": "body",
```

Comment:
- eslint: Expected object keys to be in ascending order. 'container' should be before 'emptytext'.